static/js/place/place_template.tsx

```diff
@@ -749,7 +751,15 @@ class Chart extends Component<ChartPropType, ChartStateType> {
           <div id={this.props.id} className="svg-container"></div>
           <footer className="row explore-more-container">
             <div>
-              Data from <a href={config.url}>{config.source}</a>
+              <span>Data from </span>
+              {this.state.sources.map((source, index) => {
+                return (
+                  <span key={source}>
+                    <a href={"http://" + source}>{source}</a>
```

Comment:
is this safe to do for all the sources?

Reply:
checked this from bigquery Source.domain and the provenanceDomain should be domain only without http or https

Reply:
could you add a todo, that we expect to get the full url?